docs/support/index.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: layouts/pages/basic.njk
+title: Get Support
+hasToc: false
+tags:
+  - support
+importElements:
+  - rh-tile  
+  - rh-tag
+---
+
+<link rel="stylesheet" href="{{ '/assets/packages/@rhds/elements/elements/rh-tile/rh-tile-lightdom.css' | url }}">
+<style>
+  #contact-grid h3 {
+    display: flex;
+    gap: var(--rh-space-lg, 16px);
+  }
+  #support-footer-grid {
+    margin-block-start: var(--rh-space-6xl, 96px);
+  }
+</style>
+
+## Need help?
+
+Find the best method for getting your question to the team.
+
+
+## Connect on GitHub
+
+The best way to get help or feedback is through GitHub. This allows us to pull your GitHub issue or discussion into our workflow more easily.
+
+
+### Design system help
+
+<div id="rhds-help-grid" class="grid sm-three-columns">
+  <rh-tile>
+    <h4 slot="headline"><a href="https://github.com/RedHat-UX/red-hat-design-system/issues/new?labels=bug&template=3-bug-report.yml&title=%5Bbug%5D+%3Ctitle%3E">File a bug report</a></h4>
+    If something is broken in an existing web component, submit a bug report and describe the error as thoroughly as possible.
+  </rh-tile>
+  <rh-tile>
+    <h4 slot="headline"><a href="https://github.com/RedHat-UX/red-hat-design-system/issues/new?assignees=&labels=feature,needs%20discovery&projects=&template=4-feature-request.yml&title=%5Bfeat%5D+%3Ctitle%3E">Request a feature or component</a></h4>
+    Help shape the design system’s roadmap by requesting new features or components.
+  </rh-tile>
+  <rh-tile>
+    <h4 slot="headline"><a href="https://github.com/orgs/RedHat-UX/discussions/new/choose">Start a discussion</a></h4>
+    Ask the community of Red Hat Design System users and contributors a question.
+  </rh-tile>
+</div>
+
+
+### Documentation and Figma help
+
+<div id="docs-help-grid" class="grid sm-three-columns">
+  <rh-tile>
+    <h4 slot="headline"><a href="https://github.com/orgs/RedHat-UX/discussions/new?category=documentation-feedback">Send documentation feedback</a></h4>
+    If you have general feedback about content on ux.redhat.com, start a discussion.
+  </rh-tile>
+  <rh-tile>
+    <h4 slot="headline"><a href="https://github.com/RedHat-UX/red-hat-design-system/issues/new?assignees=&labels=bug,docs&projects=&template=3-bug-report.yml&title=%5Bbug%5D+%3Ctitle%3E">Report a bug on ux.redhat.com</a></h4>
+    Submit an issue to fix bugs, like 404s, on ux.redhat.com.
+  </rh-tile>
+  <rh-tile>
+    <h4 slot="headline"><a href="https://github.com/orgs/RedHat-UX/discussions/new?category=design-library-feedback">Submit design library feedback</a></h4>
+    Create a discussion to request new features or fixes for the Red Hat Design System’s Figma library.
+  </rh-tile>
+</div>
```

Comment:
I think a reorder might help place the Bug fix first as to me that might be the most used, then followed by the feedback links.

```Suggestion
<div id="docs-help-grid" class="grid sm-three-columns">
  <rh-tile>
    <h4 slot="headline"><a href="https://github.com/RedHat-UX/red-hat-design-system/issues/new?assignees=&labels=bug,docs&projects=&template=3-bug-report.yml&title=%5Bbug%5D+%3Ctitle%3E">Report a bug on ux.redhat.com</a></h4>
    Submit an issue to fix bugs, like 404s, on ux.redhat.com.
  </rh-tile>
  <rh-tile>
    <h4 slot="headline"><a href="https://github.com/orgs/RedHat-UX/discussions/new?category=documentation-feedback">Send documentation feedback</a></h4>
    If you have general feedback about content on ux.redhat.com, start a discussion.
  </rh-tile>
  <rh-tile>
    <h4 slot="headline"><a href="https://github.com/orgs/RedHat-UX/discussions/new?category=design-library-feedback">Submit design library feedback</a></h4>
    Create a discussion to request new features or fixes for the Red Hat Design System’s Figma library.
  </rh-tile>
</div>
```

Reply:
Reordered.